pom.xml

```diff
@@ -59,6 +59,22 @@
             <distribution>repo</distribution>
         </license>
     </licenses>
+    <repositories>
```

Comment:
Can you merge the latest code of development? Code changes other than the pr function are not accepted.